gitlab-ci/src/dependencies/model/ic.py

```diff
@@ -1,11 +1,25 @@
-from typing import List
+import os
 
 from model.project import Project
 from model.repository import Repository
 from model.team import Team
 from scanner.dependency_scanner import PROJECT_ROOT
 
 IS_PRIVATE = PROJECT_ROOT.name == "ic-private"
+GITHUB_REPOSITORY = os.environ.get("GITHUB_REPOSITORY", "dfinity/ic")
```

Comment:
Can we add these variables to the CI env def of all the dependency jobs, so that its transparent what variables are needed for the job?